src/sentry/static/sentry/app/utils/deviceNameMapper.jsx

```diff
@@ -1,7 +1,7 @@
 import iOSDeviceList from 'ios-device-list';
 
 export default function(model) {
-  if (!model) {
+  if (!model || typeof model !== 'string') {
```

Comment:
We use the `isString` from `_` elsewhere it seems becuase I'm pretty sure this doesn't cover all the cases.